apps/addons/management/commands/approve_addons.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand, CommandError
+
+import amo
+from addons.models import Addon
+from editors.helpers import ReviewHelper
+
+
+class Command(BaseCommand):
+    args = '<addon_guid addon_guid ...>'
+    help = 'Approve a list of add-ons, given their GUIDs.'
+
+    def handle(self, *args, **options):
+        if len(args) == 0:  # No GUID provided?
+            raise CommandError(
+                'Please provide at least one add-on guid to approve.')
+
```

Comment:
Should we think about chunking or an upper limit, I don't know how things work now, but in the past something like a 1000 GUIDs (or maybe even 400) would be a large query. One of those things that always passes when you have a few apps, but might go wrong in larger numbers.